website/source/docs/cli/ssh.html.md

```diff
@@ -13,6 +13,82 @@ description: |-
 
 This will SSH into a running Vagrant machine and give you access to a shell.
 
+On a simple vagrant project, the instance created will be named default.
+
+Vagrant will ssh into this instance without the instance name:
+
+```bash
+$ vagrant ssh
+
+Welcome to your Vagrant-built virtual machine.
+Last login: Fri Sep 14 06:23:18 2012 from 10.0.2.2
+$ logout
+Connection to 127.0.0.1 closed.
+```
+Or you could use the name:
+```bash
```

Comment:
Add a space above this, the markdown isn't rendering for the code block.

Reply:
You might also need a space above line 28 too for markdown formatting to work...

Reply:
done